### ISSUE TITLE
Deal with different styles of measure properites on observations (multiple/single)

### ISSUE THREAD

**Issue by RicSwirrl** [Swirrl/graphql-qb#34]
If there's only a single measureType in a dataset, its legal to not specify MeasureType on the observations: http://opendatacommunities.org/data/housing-market/right-to-buy/starts-or-acquisitions/letting/2015-Q3/E92000001/starts-acqs

Note that even when there's only one measureType in the dataset, its also legal to specify MeasureType. e.g: http://statistics.gov.scot/data/fire---type-of-incident/government-year/2010-2011/S92000003/accident-status/not-accidental/type-of-fire/all/fires/count

If there are multiple measures in a dataset (you need to specify MeasureType). e.g: http://statistics.gov.scot/data/alcohol-related-discharge/government-year/1986-1987/S92000003/hospital-discharges/count

**Comment by zeginis** [comment]
This is related with one of the best practises we are proposing: https://islab-uom.github.io/qbBestPractices/#definingMultipleMeasures

**Comment by RicSwirrl** [comment]
Yeah, agree we should aim to standardise, but we want this to work for existing data too (which we might not be able to change easily)  👍
